base/Makefile

```diff
@@ -56,12 +56,21 @@ endif
 	@echo "const libllvm_version_string = \"$$($(LLVM_CONFIG_HOST) --version)\"" >> $@
 	@echo "const VERSION_STRING = \"$(JULIA_VERSION)\"" >> $@
 	@echo "const TAGGED_RELEASE_BANNER = \"$(TAGGED_RELEASE_BANNER)\"" >> $@
+ifeq ($(OS),WINNT)
+	@echo "const SYSCONFDIR = \"$(shell echo $(sysconfdir_rel) sed -e 's:/:\\\\:g')\"" >> $@
+	@echo "const DATAROOTDIR = \"$(shell echo $(datarootdir_rel) sed -e 's:/:\\\\:g')\"" >> $@
+	@echo "const DOCDIR = \"$(shell echo $(docdir_rel) sed -e 's:/:\\\\:g')\"" >> $@
+	@echo "const LIBDIR = \"$(shell echo $(libdir_rel) sed -e 's:/:\\\\:g')\"" >> $@
+	@echo "const PRIVATE_LIBDIR = \"$(shell echo $(private_libdir_rel) sed -e 's:/:\\\\:g')\"" >> $@
+	@echo "const INCLUDEDIR = \"$(shell echo $(includedir_rel) sed -e 's:/:\\\\:g')\"" >> $@
```

Comment:
Ah sorry, not sure on what you mean here. 

> or wrap the whole expression here in `'` instead of `"`
`	@echo 'const INCLUDEDIR = \"$(shell echo $(includedir_rel) sed -e 's:/:\\\\:g')\"' >> $@`
 probably not what you meant 

> use `$$` so the shell expand
`	@echo "const INCLUDEDIR = \"$(shell echo $(includedir_rel) sed -e 's:/:\\\\:g')\"" >> $@`
which part exactly?